qutebrowser/config/config.py

```diff
@@ -162,13 +163,39 @@ def get_bindings_for(self, mode: str) -> Dict[keyutils.KeySequence, str]:
                 bindings[key] = binding
         return bindings
 
+    def _implied_cmd(self, cmdline: str) -> str:
+        """Return cmdline, or the implied cmd if cmdline is a set-cmd-text."""
+        try:
+            results = runners.CommandParser().parse_all(cmdline, aliases=False)
+            if len(results) == 0:
+                return ""
```

Comment:
I'd prefer using `None` as a sentinel value, rather than an empty string (and make the type `Optional[str]`), so that it's clear that's something the caller needs to handle.